Source/Editor/Surface/Elements/OutputBox.cs

```diff
@@ -163,6 +163,12 @@ public void DrawConnections(ref Float2 mousePosition)
                     highlight += 0.5f;
                 }
 
+                // Increase thickness on impulse/ execution lines
+                if (targetBox.CurrentType.IsVoid)
+                {
+                    highlight += 1.25f;
+                }
```

Comment:
Only suggestion is that for single line if statements, Mafi likes them to be like this
```
if (targetBox.CurrentType.IsVoid)
     highlight += 1.25f;
```